library/Zend/View/Helper/Placeholder/Registry.php

```diff
@@ -39,6 +39,13 @@
  */
 class Zend_View_Helper_Placeholder_Registry
 {
+    /**
+     * View object
+     * 
```

Comment:
remove trailing whitespace

Reply:
trailing space is still there

Reply:
Now, I fixed here https://github.com/hungtrinh/zf1-future/commit/f269ec7d3b1ec1720b03862cbcef6a52d4e44464